pyproject.toml

```diff
@@ -0,0 +1,73 @@
+# SPDX-FileCopyrightText: 2024 Jev Kuznetsov, written for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+
+[build-system]
+requires = ["setuptools>=61.0", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "circup"
+dynamic = ["version"]
+description = "A tool to manage/update libraries on CircuitPython devices."
+readme = "README.rst"
+authors = [{ name = "Adafruit Industries", email = "circuitpython@adafruit.com" }]
+license = { file = "LICENSE" }
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Education",
+    "Topic :: Software Development :: Embedded Systems",
+    "Topic :: System :: Software Distribution"
+]
+keywords = ["adafruit", "blinka", "circuitpython", "micropython", "libraries"]
+
+
+dependencies = [
+    "semver~=3.0",
+    "build",
```

Comment:
Is `"build"` a new dependency? I'm not familiar with that, but I noticed it wasn't listed in the old requirements.txt.

Reply:
[build](https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives) is required for packaging. Should not be in normal deps. I'll fix that.

Reply:
is it a new requirement? Was it required in the past? I'm not familiar with it. 

I didn't see it listed in requirements or optional_requirements from before and the packaging was always working properly as far as I understand. 

Is there some other specific change within this PR that makes this module now a requirement? Or was it always required and being installed in some other way before?